dimagi/utils/couch/undo.py

```diff
@@ -32,3 +35,13 @@ def soft_delete(self, domain_included=True):
             record.save()
             self.save()
             return record
+
+
+def is_deleted(doc):
+    """
+    Guess if a document is deleted. Returns False if we're not sure.
```

Comment:
good call, that comment was terrible :)

https://github.com/dimagi/dimagi-utils/pull/300